### PR TITLE
feat: submit projects to remote contest server

### LINKED ISSUE #143: Contest server

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #143 #142 

Check out the `.env.example`

Depends on https://github.com/decentraland/builder-contest-server/pull/3

### CODE REVIEW

File: src/modules/contest/sagas.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
-import { put, takeLatest } from 'redux-saga/effects'
+import { put, takeLatest, call, select } from 'redux-saga/effects'
 
 import { SUBMIT_PROJECT_REQUEST, SubmitProjectRequestAction, submitProjectSuccess, submitProjectFailure } from './actions'
+import { getData as getProjects } from 'modules/project/selectors'
+import { getData as getScenes } from 'modules/scene/selectors'
+import { getState as getStorage } from 'decentraland-dapps/dist/modules/storage/selectors'
```
Comment (nicosantangelo) [inline src/modules/contest/sagas.ts]: TODO: This should be on the top of the file

**Comment by belohlavek** [comment]
screen shot 2019-02-15 at 13 54 19

Shouldn't this be red? (To comply with the parcel limits error and the custom layout error)

**Comment by belohlavek** [comment]
kapture 2019-02-15 at 13 56 10

Cancel button styles are inconsistent

**Comment by cazala** [comment]
> kapture 2019-02-15 at 13 56 10
>
>Cancel button styles are inconsistent

This is fixed in `master` already, I moved the styles to the `Modal.css` theme in an earlier PR, just rebase to fix it.

**Comment by nicosantangelo** [comment]
> screen shot 2019-02-15 at 13 54 19
> 
> Shouldn't this be red? (To comply with the parcel limits error and the custom layout error)

I've used the var(--error) css variable, should we change it?
@belohlavek

**Comment by cazala** [comment]
> I've used the var(--danger) css variable, should we change it?

I think there's a `var(--error)` one

**Comment by belohlavek** [comment]
Are we ok with this one? We should have a deployment for the server right?

**Comment by nicosantangelo** [comment]
> Are we ok with this one? We should have a deployment for the server right?

Yes! but it shouldn't explode if the server URL is `undefined`

**Pull request merged**